app/models/region.model.js

```diff
@@ -23,6 +23,14 @@ class RegionModel extends BaseModel {
           from: 'water.regions.region_id',
           to: 'water.licences.region_id'
         }
+      },
+      billing_batches: {
```

Comment:
Just a minor tweak. We can use camelcase for the names of the relationships and that will follow what we've already done in the `LicenceModel`.

```suggestion
      billingBatches: {
```